### ISSUE TITLE
About the loss

### ISSUE THREAD

**Issue by feiaxyt** [HelloRicky123/Siamese-RPN#14]
You did a good work. I also implemented the training code, but I always get a higher loss in the validation set compared to the training set. The performance on the OTB dataset is not good. Do you have this problem?  Whats the values of your loss of the regression and classification branch, respectively?
